### DISCUSSION TITLE
Rules playground/simulator

### DISCUSSION THREAD

**Discussion by genu** [directus/directus#15171] [Feature Requests]
It would be nice if there was a built-in rules playground similar to how firebase does it, which allows for testing rules directly in directus.

This is helpful, because as apps grow, rules can get pretty complicated.

See: https://firebase.google.com/docs/rules/simulator

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
